homeassistant/components/onewire/sensor.py

```diff
@@ -286,12 +301,13 @@ def get_entities(config):
 class OneWire(Entity):
     """Implementation of a 1-Wire sensor."""
 
-    def __init__(self, name, device_file, sensor_type):
+    def __init__(self, name, device_file, sensor_type, device_info=None):
```

Comment:
```suggestion
    def __init__(self, name, device_file, sensor_type, device_info):
```

`device_info` should always be set right ?

Reply:
It doesnt get set on OWFS (deprecated implementation)

Reply:
I was trying to avoid changes to the OWFS impementation, but hopefully @MartinHjelmare won't mind this little tweak...
He was involved on PR #42041 so maybe he has an opinion.

Reply:
Since it's currently optional it makes sense to default it to `None`.

Reply:
So should I revert 1ccd6c4 ?

Reply:
That's my advice.

Reply:
Thanks @MartinHjelmare, I've reverted the last commit.
Are you happy with that @Quentame ?

Reply:
(and how to re-trigger checks?)

Reply:
Yes, happy, that was an interrogation.
I did not see all history about that or maybe I miss something in the review, nice 👍